1-js/02-first-steps/09-comparison/1-comparison-questions/solution.md

```diff
@@ -10,12 +10,12 @@ null == "\n0\n" → false
 null === +"\n0\n" → false
 ```
 
-Some of the reasons:
```

Comment:
Please translate this line and make sure that there are no unnecessary line shifts after the translation (like added/removed new lines etc.). This helps reviewing future updates to translations.

Reply:
This is not yet resolved. `Պատճառներից որոշները․` is on line 14 but it should be on line 13 as the original text goes.

Reply:
/done